src/components/footer.js

```diff
@@ -6,8 +6,8 @@ export const Footer = () => {
 
     return (
         <div className="app_footer">
-            <div className="app_footer_item"><img alt="iRODS Icon" className="app_footer_logo" src={logo}></img><a href="https://irods.org" className="app_footer_text">iRODS Consortium © 2022</a></div>
+            <div className="app_footer_item"><img alt="iRODS Icon" className="app_footer_logo" src={logo}></img><a href="https://irods.org" className="app_footer_text">iRODS Consortium © {new Date().getFullYear()}</a></div>
             <div className="app_footer_item">Zone Management Tool Version: {version}, {process.env.REACT_APP_GIT_SHA.substring(0, 7)}</div>
         </div>
     )
-}
+}
```

Comment:
does this mean there is no longer a newline character at the end?

Reply:
I don't think there ever was a newline character at the end of the file. You can see the original footer.js [here](https://github.com/irods/irods_client_zone_management_tool/blob/main/src/components/footer.js) and my updated version [here](https://github.com/ganning127/irods_client_zone_management_tool/blob/main/src/components/footer.js).

If you're interested, this is what the footer looks like with the updated year and my new code:
<img width="1043" alt="Screenshot 2023-06-19 at 5 43 15 PM" src="https://github.com/irods/irods_client_zone_management_tool/assets/62436772/494f9fd6-cb85-44b6-8c28-747e9107b172">

Reply:
resolving, not sure what the diff tool is telling us here.